UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs

```diff
@@ -82,6 +83,14 @@ public override void Process()
 					SP.ChangeSprite(int.Parse(SerialiseData.Substring(Start, Scanning - Start)), false);
 				}
 
+				if (SerialiseData.Length > Scanning && SerialiseData[Scanning] == '#')
```

Comment:
also need to add for the SkipSection so it can skip messages properly for a sprite manipulation (yeah is Messy idk what I was thinking when making this what I was thinking when making this, probably could just replace with json serialise and deserialise object)

Reply:
Does that change seem correct?